src/tree/registries/azure/AzureTaskTreeItem.ts

```diff
@@ -41,27 +41,21 @@ export class AzureTaskTreeItem extends AzExtParentTreeItem {
     }
 
     public hasMoreChildrenImpl(): boolean {
-        return !!this._nextLink;
```

Comment:
Paging is gone which I'm not excited about. I want to fix that.

Reply:
I opened https://github.com/Azure/azure-sdk-for-js/issues/20380 to address the lack of real paging support.